### PR TITLE
testando como funfa

### PR DESCRIPTION
**Pull request by biscoitoxoco**
## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
donde esta la biblioteca?

**Comment by optimumtact** [comment]
anyway, wrong repository?

**Comment by cacogen** [comment]
speed merge

File: yo.txt.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+yo!
```
Comment (wesoda25) [inline yo.txt.txt]: hi

**Review by iamafsh — APPROVED** [review]

File: yo.txt.txt (hunk 1 above)
Comment (OverlyGeneric) [reply yo.txt.txt]: yo, angelo.

Comment (PKPenguin321) [reply yo.txt.txt]: 🗿

**Pull request closed without merging**